docs/templates/eap-cd-amq-s2i.adoc

```diff
@@ -218,13 +218,13 @@ for more information.
 |Deployment |Variable name |Description |Example value
 
 .23+| `${APPLICATION_NAME}`
-|`MQ_SERVICE_PREFIX_MAPPING` | -- | `${APPLICATION_NAME}-amq=MQ`
-|`MQ_JNDI` | JNDI name for connection factory used by applications to connect to the broker, e.g. java:/ConnectionFactory | `${MQ_JNDI}`
+|`MQ_SERVICE_PREFIX_MAPPING` | -- | `${APPLICATION_NAME}-amq7=MQ`
+|`MQ_JNDI` | JNDI name for connection factory used by applications to connect to the broker, e.g. java:/DefaultConnectionFactory | `${MQ_JNDI}`
```

Comment:
Shouldn't this be `java:jboss/DefaultJMSConnectionFactory` ?

Reply:
Yes :). I have a final patch that actually uses this in the messaging script.